MojeWidelo_WebApi/Controllers/CommentController.cs

```diff
@@ -170,5 +170,64 @@ public async Task<IActionResult> AddCommentResponse([Required] string id)
 
 			return StatusCode(StatusCodes.Status200OK, "Odpowiedź dodana pomyślnie.");
 		}
+
+		[HttpGet("comment/commentById", Name = "getCommentById")]
```

Comment:
```suggestion
		[HttpGet("comment/commentById", Name = "getCommentById")]
		[HttpGet("comment/responseById", Name = "getResponseById")]
```

Widzę że GetCommentById i GetResponseById mają praktycznie tą samą logike, więc można do jednego kontrolera dwie ścieżki przydzielić i nie powtarzać kodu defacto